### PR TITLE
Synchronised map and gallery in sidebar

### PR DESCRIPTION
**Pull request by alan-wu**
This updates contain the following changes:

- Synchronised 2D flatmap and 3D whole body map
- Sidebar with gallery (https://github.com/ABI-Software/map-sidebar/pull/34 and https://github.com/ABI-Software/mapcore-gallery/pull/4)
- Local search on flatmap
- Changes highlight and select colour on scaffold
- Initial region tree supports on scaffold viewer
- The bar on the split panels is now its own component

### CODE REVIEW

File: src/store/modules/splitFlow.js [hunk 1]
```diff
@@ -104,7 +139,141 @@ const mutations = {
...
+        state.activeView = payload.layout;
+        state.globalCallback = true;
+      } else {
+        state.activeView = "singlepanel";
+        state.syncMode = false;
+        state.globalCallback = false;
+      }
+    }
+  },
+  closeSlot(state, payload) {
```
Comment (Tehsurfer) [inline src/store/modules/splitFlow.js:166]: Is this for closing a pane of the split pane or for removing one of the dynamic components?

**Review by Tehsurfer — APPROVED** [review]
Looks good, I couldn't find that css file anywhere and didn't understand some of it but running it it works well

File: src/store/modules/splitFlow.js (hunk 1 above)
Comment (alan-wu) [reply src/store/modules/splitFlow.js:166]: It is for closing a pane for the split pane. This function reassign the entries to different slots if they are affected by the closing slot. There is another function for removing an entry thus removing an instance of the content vuer.

File: src/assets/header-icon.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
+.tooltip {
```
[the remark this answers is not in the document]
Comment (alan-wu) [reply src/assets/header-icon.scss:1]: Yes, they are - https://github.com/alan-wu/mapintegratedvuer/blob/41431447be1d7df9724a47166137d72d8c61668b/src/components/DialogToolbarContent.vue#L280
There are other places that use those classes defined there.

**Pull request merged**